scss/_patterns_tabs.scss

```diff
@@ -71,18 +71,17 @@
       // Display the highlight when focussing in modern browsers that support
       // focus-visible.
       &:focus:not(:focus-visible) {
-        @include vf-highlight-bar($color-tabs-active-bar, bottom, false, $sph-inner);
+        @include vf-highlight-bar($color-tabs-active-bar, bottom, false, 0);
       }
 
       &:active,
-      &:hover,
       &[aria-selected='true'] {
-        @include vf-highlight-bar($color-tabs-active-bar, bottom, false, $sph-inner);
+        @include vf-highlight-bar($color-tabs-active-bar, bottom, false, 0);
 
         // Display the highlight when focussing (in combination with the parent
         // states) in modern browsers that support focus-visible.
         &:focus:not(:focus-visible) {
-          @include vf-highlight-bar($color-tabs-active-bar, bottom, false, $sph-inner);
```

Comment:
As mentioned in other comment, these 0s at the end param can probably be removed, and I guess this padding param is not used anywhere anymore, so could be removed from code completely.